Cython/Includes/cpython/mem.pxd

```diff
@@ -22,6 +24,15 @@ cdef extern from "Python.h":
     # for the I/O buffer escapes completely the Python memory
     # manager."
 
+    IF PY_VERSION_HEX >= 0x03040000:
```

Comment:
No, that's not how this works. The Python version only comes into play at C compilation time, not during the C code generation. 

Instead, add the fallback code to `ModuleSetupCode.c` and always define the functions here.

Reply:
Ok, thanks. Knew something was off, but wasn't sure how to fix it. Will update.